packages/list/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rimbu/list",
-  "version": "0.12.2",
+  "version": "0.13.0",
```

Comment:
We have lerna to automatically do version increments for us, so no need to manually increase the version number